packages/server/graphql/public/mutations/acceptTeamInvitation.ts

```diff
@@ -0,0 +1,127 @@
+import toTeamMemberId from 'parabol-client/utils/relay/toTeamMemberId'
+import {InvitationTokenError, SubscriptionChannel} from '../../../../client/types/constEnums'
+import AuthToken from '../../../database/types/AuthToken'
+import acceptTeamInvitationSafe from '../../../safeMutations/acceptTeamInvitation'
+import {getUserId, isAuthenticated} from '../../../utils/authorization'
+import encodeAuthToken from '../../../utils/encodeAuthToken'
+import publish from '../../../utils/publish'
+import RedisLock from '../../../utils/RedisLock'
+import segmentIo from '../../../utils/segmentIo'
+import activatePrevSlackAuth from '../../mutations/helpers/activatePrevSlackAuth'
+import handleInvitationToken from '../../mutations/helpers/handleInvitationToken'
+import {MutationResolvers} from '../resolverTypes'
+import getIsUserIdApprovedByOrg from './helpers/getIsUserIdApprovedByOrg'
+
+const acceptTeamInvitation: MutationResolvers['acceptTeamInvitation'] = async (
+  _source,
+  {invitationToken, notificationId},
+  {authToken, dataLoader, socketId: mutatorId}
+) => {
+  // AUTH WORKAROUND
+  if (!isAuthenticated(authToken) || !invitationToken) {
+    // Workaround for https://github.com/zalando-incubator/graphql-jit/issues/171
+    return {}
```

Comment:
found a bug in graphql-jit. this return value is the same thing as if `@include(if: false)` directive worked correctly